diff --git a/node_modules/@modelcontextprotocol/inspector/client/dist/assets/index-Bl2jgoAl.js b/node_modules/@modelcontextprotocol/inspector/client/dist/assets/index-Bl2jgoAl.js
index c245baf..900ed4d 100644
--- a/node_modules/@modelcontextprotocol/inspector/client/dist/assets/index-Bl2jgoAl.js
+++ b/node_modules/@modelcontextprotocol/inspector/client/dist/assets/index-Bl2jgoAl.js
@@ -17082,7 +17082,7 @@ const oauthTransitions = {
       context.provider.saveServerMetadata(parsedMetadata);
       context.updateState({
         resourceMetadata,
-        resource,
+        resource: undefined, // TODO: This must be undefined for azure
         resourceMetadataError,
         authServerUrl,
         oauthMetadata: parsedMetadata,
@@ -17118,6 +17118,13 @@ const oauthTransitions = {
     execute: async (context) => {
       const metadata = context.state.oauthMetadata;
       const clientInformation = context.state.oauthClientInfo;
+
+      // Azure supports PKCE, but it's not advertised in metadata
+      if (metadata.issuer.includes("login.microsoftonline.com") && !metadata.code_challenge_methods_supported) {
+        console.log("Detected Azure AD - adding PKCE support to metadata");
+        metadata.code_challenge_methods_supported = ["S256"];
+      }
+
       const scope = await discoverScopes(
         context.serverUrl,
         context.state.resourceMetadata ?? void 0
@@ -17130,7 +17137,7 @@ const oauthTransitions = {
           redirectUrl: context.provider.redirectUrl,
           scope,
           state: generateOAuthState(),
-          resource: context.state.resource ?? void 0
+          // resource: context.state.resource ?? void 0
         }
       );
       context.provider.saveCodeVerifier(codeVerifier);
